app/mci/cmd/create.go

```diff
@@ -173,25 +177,34 @@ func getClusters(kubeconfig string) ([]string, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error in getting contexts from kubeconfig: %s", err)
 	}
+	glog.Infof("contexts found:\n%v", output)
 	return strings.Split(output, "\n"), nil
 }
 
 // Creates the given ingress in the given list of clusters.
 func createIngressInClusters(kubeconfig, ingressFilename string, clusters []string) error {
-	kubectlArgs := []string{"kubectl"}
-	if kubeconfig != "" {
-		kubectlArgs = append(kubectlArgs, fmt.Sprintf("--kubeconfig=%s", kubeconfig))
+	var ing v1beta1.Ingress
+	if err := unmarshall(ingressFilename, &ing); err != nil {
+		return fmt.Errorf("error in unmarshalling the yaml file %s, err: %s", ingressFilename, err)
 	}
+	glog.V(2).Infof("Unmarshaled this ingress:\n%+v", ing)
```

Comment:
I generally use V(5) for printing objects and V(2) for shorter messages like ("unmarshalling of ingress successful").